src/Maestro/Maestro.Web/Controllers/AzDevController.cs

```diff
@@ -46,5 +47,20 @@ public async Task<IActionResult> GetBuildStatus(string account, string project,
                         Convert.ToBase64String(Encoding.UTF8.GetBytes(":" + token)));
                 });
         }
+
+        [HttpGet("getFile/{account}/{project}/{repo}/{*filePath}")]
```

Comment:
I assume that this code is getting the Versions.Details file? Or is it doing something else?

Reply:
It is and it should have been pulled out with the other stuff that does that. Thanks for the catch! :)